plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSourceProvider.java

```diff
@@ -448,7 +461,9 @@ private TupleDomain<IcebergColumnHandle> prunePredicate(
         }
         // Filter out partition columns domains from the dynamic filter because they should be irrelevant at data file level
         return unenforcedPredicate
-                .filter((columnHandle, domain) -> !partitionKeys.containsKey(columnHandle.getId()));
+                .filter((columnHandle, domain) -> !partitionKeys.containsKey(columnHandle.getId()))
```

Comment:
Move the comment 

```
// Filter out partition columns domains from the dynamic filter because they should be irrelevant at data file level
```

above this line